docs/doc_sources/beginners_guides/installation.rst

```diff
@@ -97,14 +97,15 @@ Directions to install drivers for Intel GPU devices are available at https://dgp
 
 Once ``dpctl`` is installed, use ``python -m dpctl --full-list`` to list recognized devices.
 
-For ``dpctl`` to target Intel GPU devices, appropriate drivers should be installed systemwide.
+For ``dpctl`` to target Intel GPU devices, appropriate drivers should be installed system wide.
```

Comment:
According to https://www.calstate.edu/csu-system/csu-branding-standards/editorial-style-guide/Pages/Commonly-Used-Words.aspx#W

It should be 'systemwide', because system has two syllables. I looked it up when writing this text.

Reply:
I don't often see it spelled that way, but your point is valid. I changed it back to systemwide.